docker-files/home/.irbrc

```diff
@@ -3,9 +3,7 @@
 require 'irb/completion'
 require 'irb/ext/save-history'
 
-ARGV.concat ['--readline',
```

Comment:
Which cop reported an offense for this?

Reply:
This one https://github.com/rubocop/rubocop/issues/11277

Reply:
I'll reword the commit to reflect that